src/app/modules/scheduler/components/scheduler-modal/scheduler-date-examples/scheduler-date-examples.component.html

```diff
@@ -1,5 +1,5 @@
 <div *ngFor="let date of scheduleExamples | slice: 0:maxExamples" class="schedule-example">
-  {{ date | formatDateTime: null }}
+  {{ date | formatDateTime }}
```

Comment:
1. Not due to this PR, but I cannot test this as scheduler modal won't open.
2. This looks like this is going to show timestamps in browser timezone. In this component this is incorrect as we want to show dates in server timezone.
I would suggest adding another `serverTime` directive to be used like so `date | serverTime | formatDateTime`.
I have not checked other places, but other places like Audit and Reporting look suspicious.